examples/jupyter/README.md

```diff
@@ -0,0 +1,18 @@
+# Jupyter 
+It is rather simple to deploy Jupyter on MANTL. There are some official Docker images that you can use for this purpose (https://github.com/jupyter/docker-stacks). For this example we use the jupyter/minimal-notebook.
+
+
+We wrapped the Marathon submit REST call in a small script: `marathon_submit.sh`. You can use it to deploy Jupyter on your MANTL cluster running the following commands.
+
+```bash
+source bin/set_env.sh #you will asked to enter your control node hostname (without https://), and admin password 
+bin/marathon_submit.sh jupyter.json
+```
+
+If everything went fine, you should be able to figure out the front end URL of your Jupyter deployment from the Traefik UI.
```

Comment:
You get it in the Traefik UI homepage. I had a workshop today with roughly 20 people (https://github.com/phnmnl/workflow-demo), everyone managed to figure it out :grinning: